Gemfile.lock

```diff
@@ -166,7 +166,8 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.6)
+    mimemagic (0.3.8)
```

Comment:
They've released 0.3.9 with some fixes.

Here's the [changelog](https://github.com/mimemagicrb/mimemagic/blob/master/CHANGELOG.md#039-2021-03-25):
* Resolve issues parsing the version of freedesktop.org.xml shipped with Ubuntu Trusty.
* Reintroduce overlays, since it seems at least some people were using them.
* Make Rake a runtime dependency.
* Fix the test suite.

We might be fine with the version 0.3.8.

Reply:
This fiasco is probably not over yet, and we may be forced to upgrade to 0.3.9 for all we know :sob:

Reply:
Oops :P I see we did update to 0.3.9. Nevermind!

Reply:
Yup, forgot to mention that I've already done it :see_no_evil:

Reply:
Thank you for pushing to my PR. I really love this feature and think we should use it more often to move the work along faster between time zones when it makes sense. :+1: